_projects/2022/100470958/100470958.Rmd

```diff
@@ -0,0 +1,389 @@
+---
+title: "Satellite images defy economic data of authoritarian regimes"
+description: |
+  Highly customized dumbbell chart that compares official GDP against satellite-based estimation of GDP, across 103 countries.
+categories: "2022"
+author: Freddy Andrés Camargo
+date: "`r Sys.Date()`"
+citation: Camargo, 2023
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE, warning=FALSE}
+knitr::opts_chunk$set(warning = FALSE, message = FALSE) 
+```
+
+Satellite images of the brightness of the earth, also called nighttime lights data (NTL, from the US Air Force’s Defense Meteorological Satellite Program), have been used as a proxy for economic activity. Luis R. Martínez, professor at the Harris School of Public Policy at the University of Chicago, not only developed a model to calculate the GDP of every country based in the NTL data set, but he also found that countries with less political rights and civil liberties report overestimated GDP. The goal of this article is to rebuild a chart designed by The Economist to show the magnitude of this finding.
+
+![The Economist's chart](the_economist_gdp_growth_estimates.png "The Ecnomist's chart")
+
+## Data collection
+
+The data includes different records per country: the GDP from World Bank, the GDP estimated based on NTL, and the yearly growth of each one between 2002 and 2021. In addition, it includes the status in the Freedom in the World Index 2021, published by Freedom House, which classifies each country in three categories (Free, Partly Free, and Not Free). Finally, the population is incorporated, because for visualization purposes the chart only shows countries with more than 5 million population. Mr. Martínez provided me the projections of the NTL model, after contacting him by email.
+
+```{r}
+library(ggplot2)
+library(ggthemes)
+library(ggrepel)
+library(tidyverse)
+library(patchwork)
+```
+Let's see how the data looks like:
+
+```{r include=FALSE}
```

Comment:
Except for the initial setup, please do not hide any chunk.

Reply:
ok, done